src/utils/utils.js

```diff
@@ -29,3 +29,11 @@ export const uniqueArray = (arr) => {
   }
   return result
 }
+
+export const cloneNode = (node) => {
+  const dupNode = node.cloneNode(true);
+  ['onclick', 'onmouseover', 'onmouseout'].forEach(function (method) {
```

Comment:
- What happens for the other events?
 - Could you also explain the need for cloning?
 - `function (method) { ... }` => `(method) => dupNode[method] = node[method]`